packages/button/src/styled/Button.ts

```diff
@@ -62,7 +63,7 @@ export const Postfix = styled(PrefixPostfixBase)`
 ) => buttonAppearanceTheme(theme, appearance, baseAppearance, 'postfixSpacing')}px;
 `;
 
-export const ContentWrapper = styled.span`
+export const ContentWrapper = styled.span<IContentProps>`
```

Comment:
You need to toss appearance and baseAppearance inside this component and inside Content.